tests/app/questionnaire/routing/test_operators.py

```diff
@@ -314,6 +315,64 @@ def test_operation_date(date_string: str, offset):
     assert operator.evaluate(operands) == expected_result
 
 
+@pytest.mark.parametrize(
+    "offset, expected_result",
+    [
+        # Last Thursday (Yesterday from reference date)
```

Comment:
I find these comments confusing. I think we need to rationalise reference date and Initial date (used in doc string) and move it to the top, because as soon as I see the first comment "Yesterday from reference date", I am looking for what it is. Yesterday is also a funny term used in this context, it should be "day before reference date". As a soon as you say yesterday I am thinking today.